internal/resources/resource_lb_virtual_servers.go

```diff
@@ -0,0 +1,139 @@
+// (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+
+package resources
+
+import (
+	"context"
+
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/internal/resources/validations"
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/internal/utils"
+	"github.com/HewlettPackard/hpegl-vmaas-terraform-resources/pkg/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func LoadBalancerVirtualServers() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"vip_name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "vip_name of Network loadbalancer virtual server name",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "description of Network loadbalancer virtual server",
+				ForceNew:    true,
+			},
+			"vip_address": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "vip_address of Network loadbalancer virtual server",
+				ForceNew:    true,
```

Comment:
if the user changes this, it will create a new virtual server ?

Reply:
done